templates/forms/_talk.twig

```diff
@@ -56,6 +57,7 @@
                 <option value="entry" {% if level == 'entry' %}selected {% endif %}>Entry level</option>
                 <option value="mid" {% if level == 'mid' %}selected {% endif %}>Mid-level</option>
                 <option value="advanced" {% if level == 'advanced' %}selected {% endif %}>Advanced</option>
+                <option value="mixed" {% if level == 'mixed' %}selected {% endif %}>Soft Skills/All levels</option>
```

Comment:
This is something I've heard people talk about a lot being a really odd question and not applicable. Your call of course but I think people will appreciate it.